static/src/stylesheets/module/_adslot.scss

```diff
@@ -243,7 +243,6 @@
         position: relative;
         height: auto;
         margin: 0 $gs-gutter/2;
-        width: gs-span(4.5);
```

Comment:
So maybe you should remove that line in another pr? In case of roll out it won't affect whole functionality here.